Source/Resources/Content/MSGraph/ConditionalAccess/Policies/Baseline - Block High Risk Sign-Ins.json

```diff
@@ -0,0 +1,49 @@
+{

```

Comment:
how do these CA policies overlap with our existing ones? are there ones we should remove?